codereadyworkspace/hogarama_local.env

```diff
@@ -24,3 +24,15 @@ export AMQ_URL=ssl://localhost:8883
 ##  KEYCLOAK  ##
 ################
 export KEYCLOAK_AUTH_SERVER_URL=https://keycloak-gepardec.apps.p.aws.ocp.gepardec.com/auth/
+
+################
+##  KAFKA     ##
+################
+#export KAFKA_CONNECTION=localhost:9092
+export KAFKA_CONNECTION=https://gepardec-kafka-cluster-kafka-bootstrap-kafka.apps.p.aws.ocp.gepardec.com:443
+export KAFKA_TOPIC_PREFIX="erhard."
```

Comment:
Vllt sollten wir hier einen allgemeinerern perfix fuer den master branch verwenden

Reply:
last week, we briefly talked about this issue... when multiple devs are using the same topic in AWS (at the dev-stage), they all produce to and consume from the same topic. one possible solution is a personal prefix, so each dev has its personal topic.

still, hard coding @ErhardSiegl 's personal prefix is pretty funny...

Reply:
Sounds good to me.